### ISSUE TITLE
Add methods for Issue Lists

### ISSUE THREAD

**Issue by jack-lewin** [jack-lewin/node-donedone-api#4]
The following API methods need implementing:

- [ ] Issues you're waiting on
- [ ] Issues you're CC'd on
- [ ] Your active issues
- [ ] All your issues
- [ ] All active issues
- [ ] All closed & fixed issues
- [ ] All issues on hold
- [ ] All issues
- [ ] Issues by custom filter
